docs/documentation/bundles/alpha.rst

```diff
@@ -12,15 +12,35 @@ to change prior to final release and in most cases are missing documentation.
 
 - `invenio-accounts-rest <https://invenio-accounts-rest.readthedocs.io>`_
     - REST APIs for account management.
+- `invenio-app-rdm <https://inveniordm.docs.cern.ch>`_
+    - Research Data Management (RDM) flavour of Invenio
+- `invenio-app-ils <https://invenio-app-ils.readthedocs.io>`_
+    - Integrated Library System (ILS) flavour of Invenio.
+- `invenio-banners <https://invenio-banners.readthedocs.io>`_
+    - Create and show banners with useful messages to users.
 - `invenio-charts-js <https://invenio-charts-js.readthedocs.io>`_
     - AngularJS application for producing charts.
+- `invenio-circulation <https://invenio-circulation.readthedocs.io>`_
+    - Library circulation module for Invenio.
+- `invenio-cli <https://inveniordm.docs.cern.ch/reference/cli/>`_
+    - CLI module for Invenio
+- `invenio-communities <https://invenio-communities.readthedocs.io>`_
+    - Invenio communities module.
 - `invenio-csl-js <https://invenio-csl-js.readthedocs.io>`_
     - AngularJS application for rendering citation strings via the records
       REST API and the CSL REST API.
 - `invenio-csl-rest <https://invenio-csl-rest.readthedocs.io>`_
     - REST API for retrieving Citation Style Language (CSL) style files.
+- `invenio-drafts-resources <https://invenio-drafts-resources.readthedocs.io>`_
+    - Submission/deposit module for Invenio.
 - `invenio-github <https://invenio-github.readthedocs.io>`_
     - GitHub integration with automatic archiving of new releases in Invenio.
+- `invenio-marc21 <https://invenio-marc21.readthedocs.io>`_
```

Comment:
This one should be left out. (I've archived the repository now)

Reply:
`invenio-marc21` removed